lib/pelemay.ex

```diff
@@ -40,7 +40,7 @@ defmodule Pelemay do
     Db.init()
 
     functions
-    |> SumMag.map(&Optimizer.replace_expr(&1))
+    |> SumMag.map(&Optimizer.replace_expr(&1, ("Elixir.PelemayNif" <> (__MODULE__ |> Generator.module_replaced_non())) |> String.to_atom()))
```

Comment:
An alternative expression instead of `__MODULE__` that expresses a caller module of the macro is better.
I can't find this expression.